index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <nav>
+        <h3 class="logo">My Logo</h3>
+        <img class="onion" src="image/Union.png" alt="" />
+    </nav>
+
+
+    <div class="container">
+
+        <section class="backgroundimg">
```

Comment:
- [ ] According to the Figma design, this section has a background image as shown below. Kindly implement it

- [ ] Currently, your design is not very well aligned with the Figma design. You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- [ ] 
Figma design: 
![](https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/headline-mobile.png)

Your design:
![image](https://user-images.githubusercontent.com/84156714/185690358-917eec17-6bdf-49ba-bcbb-96ddc8cde7eb.png)